### DISCUSSION TITLE
external tomography file usage

### DISCUSSION THREAD

**Discussion by DanBrox** [SPECFEM/specfem3d#1586] [SPECFEM2D General]
Trying to use an external 2D velocity model in SPECFEM2D by setting MODEL = external in the Par file.  When I use the attached tomo_file.xyz

[tomo_file.txt](https://github.com/SPECFEM/specfem3d/files/10854513/tomo_file.txt)

I get simulation results where the P-wave velocity appears to be larger than anticipated:

![proc000000_vp](https://user-images.githubusercontent.com/46611376/221971646-277f7ea0-c610-4cee-a4a5-152fa44175ac.jpg)

Also, when I try plotting the velocity model using the 'Visualizations' tool, it shows the following min/max P-wave velocity information which doesn't seem to agree with the tomo_file.xyz.  Suggestions as to what the problem is?

python3 plot_model_from_gll_or_binary_file.py proc000000_vp.bin
input file:  proc000000_vp.bin

reading in file:  proc000000_vp.bin

model:  proc000000_vp
  values min/max =  13866.453 / 13866.883

reading in position files...
proc000000_x.bin
proc000000_z.bin

mesh dimensions: x min/max =  0.0 / 6000.0
                 z min/max =  0.0 / 4000.0

plotting model...
  plotted as  proc000000_vp.jpg

done

**Comment by danielpeter** [comment]
@DanBrox  please follow up on this issue https://github.com/SPECFEM/specfem2d/issues/1175#issue-1611595991
